cnf-certification-test/operator/suite.go

```diff
@@ -317,3 +347,100 @@ func testOperatorSingleCrdOwner(check *checksdb.Check, env *provider.TestEnviron
 
 	check.SetResult(compliantObjects, nonCompliantObjects)
 }
+
+func testOperatorPodsRunAsUserID(check *checksdb.Check, env *provider.TestEnvironment) {
+	var compliantObjects []*testhelper.ReportObject
+	var nonCompliantObjects []*testhelper.ReportObject
+
+	for csv, pods := range env.CSVToPodListMap {
+		check.LogInfo("Csv: %q", csv)
+		for _, pod := range pods {
+			check.LogInfo("Testing Pod %q", pod)
+			if pod.IsRunAsUserID(0) {
+				check.LogError("Pod %q UserID of the pods created by the operator is 0", pod.Name)
+				nonCompliantObjects = append(nonCompliantObjects, testhelper.NewPodReportObject(pod.Namespace, pod.Name, "Pod has been found with UserID is 0", false))
+			} else {
+				check.LogInfo("Pod %q UserID of the pods created by the operator is not 0", pod.Name)
+				compliantObjects = append(compliantObjects, testhelper.NewPodReportObject(pod.Namespace, pod.Name, "Pod has been found with UserID is not 0", true))
+			}
+		}
+	}
+	check.SetResult(compliantObjects, nonCompliantObjects)
+}
+
+func testOperatorPodsRunAsNonRoot(check *checksdb.Check, env *provider.TestEnvironment) {
+	var compliantObjects []*testhelper.ReportObject
+	var nonCompliantObjects []*testhelper.ReportObject
+
+	for csv, pods := range env.CSVToPodListMap {
+		check.LogInfo("Csv: %q", csv)
+		for _, pod := range pods {
+			// We are looking through both the containers and the pods separately to make compliant and non-compliant objects.
+
+			for _, c := range pod.Containers {
+				if c.IsContainerRunAsNonRoot() {
+					check.LogInfo("Container %q created by the operator is run as not root", c.Name)
+					compliantObjects = append(compliantObjects, testhelper.NewPodReportObject(c.Namespace, c.Name, "Container has been found is run as not root", true))
+				} else {
+					check.LogError("Pod %q created by the operator is run as root", pod.Name)
+					nonCompliantObjects = append(nonCompliantObjects, testhelper.NewPodReportObject(pod.Namespace, pod.Name, "Container has been found is run as root", false))
```

Comment:
```suggestion
					check.LogError("Container %q created by the operator is run as root", pod.Name)
					nonCompliantObjects = append(nonCompliantObjects, testhelper.NewPodReportObject(pod.Namespace, pod.Name, "Container has been found is run as root", false))
```

Reply:
I changed all the messages, it is an old version.